### PR TITLE
Allow to build without distutils

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by PeterMeisrimelModelon** [comment]
Is this ready to be reviewed? I held off since it was still marked as "Draft". (I suppose same for https://github.com/modelon-community/PyFMI/pull/236)

**Comment by jschueller** [comment]
yes, this is ready: the skbuild/cmake path is not good enough because it does not allow to fully cross-compile the windows fortran modules with the 2 different compiler/linkers needed (msvc+msys/gfortran) like its currently done in conda-forge
instead I propose to use setuptools instead of distutils and ignore the fortran solvers for now with python>=3.12
it turns out we dont require them for what we do with pyfmi, but I suggest they get migrated to C in the long term (most are f77 and can be converted using f2c)

File: setup.py [hunk 1]
```diff
@@ -63,7 +73,6 @@ def remove_prefix(name, prefix):
 logging.basicConfig(level=getattr(logging,args[0].log),format='%(levelname)s:%(message)s',filename=args[0].log_file)
 logging.debug('setup.py called with the following optional args\n %s\n argument parsing completed.',vars(args[0]))
 try:
-    from subprocess import Popen, PIPE
     _p = Popen(["svnversion", "."], stdout=PIPE)
     revision = _p.communicate()[0].decode('ascii')
```
Comment (modelonrobinandersson) [inline setup.py:77]: If you don't mind you could remove this entire try/except, it is an old artifact from before assimulo moved to git.

**Review by modelonrobinandersson — APPROVED** [review]
Looks good here too, just a minor comment, and add a changelog entry, thanks again!

File: setup.py (hunk 1 above)
Comment (jschueller) [reply setup.py]: ok

**Comment by jschueller** [comment]
done

**Review by modelonrobinandersson — APPROVED** [review]

**Review by PeterMeisrimelModelon — APPROVED** [review]

**Comment by jschueller** [comment]
hello @PeterMeisrimelModelon @modelonrobinandersson could we tag a new version with this ?

**Comment by modelonrobinandersson** [comment]
> hello @PeterMeisrimelModelon @modelonrobinandersson could we tag a new version with this ?

hi @jschueller I will get back tomorrow, I am just going to make a minor test on my end with these changes and then I hope to merge it.

**Pull request merged**